bicep/aks-cluster.bicep

```
@@ -0,0 +1,75 @@
+param dnsPrefix string
+param linuxAdminUsername string
+@secure()
+param sshRSAPublicKey string
+param subnetId string
+param location string = resourceGroup().location
+
+resource aks 'Microsoft.ContainerService/managedClusters@2023-01-01' = {
+  name: 'pocimpinfak1401'
+  location: location
+  identity: {
+    type: 'SystemAssigned'
+  }
+  properties: {
+    dnsPrefix: dnsPrefix
+    kubernetesVersion: '1.32'
+
+    agentPoolProfiles: [
+      // System node pool
+      {
+        name: 'systemnp'
+        vmSize: 'Standard_DS2_v2'
+        osType: 'Linux'
+        type: 'VirtualMachineScaleSets'
+        mode: 'System'
+        vnetSubnetID: subnetId
+        enableNodePublicIP: false
+        minCount: 1
+        maxCount: 3
+        enableAutoScaling: true
+      }
+
+      // User/worker node pool
+      {
+        name: 'workernp'
+        vmSize: 'Standard_DS2_v2'
```

Can we change these to a memory-optimised SKU? (bonus: v6 VMs are now available)

```suggestion
        vmSize: 'Standard_E16as_v6'
```